call-home/Cargo.toml

```diff
@@ -2,8 +2,15 @@
 name = "call-home"
 version = "0.1.0"
 edition = "2021"
-description = "call home"
+description = "Call-home for observability"
+authors = [
+  "Vivek Kumar <vivek.kumar@datacore.com>",
+  "Niladri Halder <niladri.halder26@gmail.com>"
```

Comment:
nit: it's no longer idiomatic to add authors to rust crates and may even be removed in future versions.